next/docs/common/entities.ts

```diff
@@ -9,6 +9,7 @@ export interface NavigationEntry {
 }
 
 export enum NavigationKey {
+  HELP = "help",
```

Comment:
@esthermcdade, would you mind making sure the `NavigationKey` is arranged alphabetically. Let me know if you see an error in VSC.